components/settings/Handler.qml

```diff
@@ -53,6 +53,7 @@ Item {
         'buttonGuide': false,
         'twelveHour': false,
         'smallFont': false,
+        'preferBox': false,
```

Comment:
I prefer to default to the previous behavior when making a new option that is really a user preference. I'm OK with this change in general but I think this line should be defaulted to true.